weather_mv/loader_pipeline/regrid_test.py

```diff
@@ -32,20 +33,22 @@
 
 def make_skin_temperature_dataset() -> xr.Dataset:
     ds = xr.DataArray(
-        np.zeros((5, 6)) + 300.,
+        np.zeros((4, 5, 6)) + 300.,
```

Comment:
Consider np.full here. This seems a bit awkward.

Reply:
Thanks!